site.yml

```diff
@@ -5,3 +5,10 @@
   tags: ['common']
   roles:
     - role: common
+
+- name: Install and configure znc
+  hosts: all
```

Comment:
I would recommend using hostgroups (see #4 for an example). This would allow for expansion to multiple hosts (obviously not needed now, but maybe helpful in the future).